src/ioctls/vcpu.rs

```diff
@@ -620,7 +620,7 @@ impl VcpuFd {
     ///         userspace_addr: load_addr as u64,
     ///         flags: 0,
     ///     };
-    ///     vm.set_user_memory_region(mem_region).unwrap();
+    ///     unsafe { vm.set_user_memory_region(mem_region).unwrap() };
```

Comment:
For maximum neatness, the `unwrap` here should've been outside of the `unsafe` block. This occurs in a number of other places as well :-s